dali/c_api/c_api.cc

```diff
@@ -28,20 +28,48 @@ void daliCreatePipeline(daliPipelineHandle* pipe_handle,
     int batch_size,
     int num_threads,
     int device_id,
-    int prefetch_queue_depth) {
+    bool separated_execution,
```

Comment:
Maybe you can just use cpu_prefetch_queue_depth and gpu_prefetch_queue_depth to determine whether you should use asymmetric queue depth or not?
Then I think prefetch_queue_depth and separated_execution won't be necessary

Reply:
Not really, Pipeline with queue depth {2,2} is different from Pipeline with queue depth 2 (that's how current python API works). The former uses separate queue policy while the latter uses the uniform queue policy